plugins/org.eclipse.sirius.common.acceleo.aql.ide/build.properties

```diff
@@ -22,3 +22,4 @@ bin.includes = META-INF/,\
                about.ini
 javacProjectSettings = true
 additional.bundles = org.eclipse.core.runtime
+javacDefaultEncoding.. = UTF-8
```

Comment:
Is it normal to have .. at the end of javacDefaultEncoding?
Otherwise, it is approved

Reply:
It's what the Eclipse quick fix produces.
I think this is to allow setting different encondings for different libraries (`javacDefaultEncoding.lib.jar`) and just `..` means "everything" or something like that (`*` is probably not valid in a properties key).